protos/platform/v0/platform.proto

```diff
@@ -9,6 +9,8 @@ service Platform {
   rpc getDocuments (GetDocumentsRequest) returns (GetDocumentsResponse);
   rpc getIdentityByFirstPublicKey (GetIdentityByFirstPublicKeyRequest) returns (GetIdentityByFirstPublicKeyResponse);
```

Comment:
Let's mark them as deprecated https://stackoverflow.com/questions/43275832/how-to-mark-rpc-as-deprecated